src/components/CommentField.js

```diff
@@ -83,28 +85,30 @@ export default class CommentField extends React.Component<Props, State> {
 
   render() {
     return (
-      <form onSubmit={this.onSubmitForm} className="raf-comment-field">
+      <form onSubmit={this.onSubmitForm} className="send-comment">
         {this.props.image ? (
           <Avatar image={this.props.image} circle size={39} />
         ) : null}
-        <div className="raf-comment-field__group">
+        <div className="d-flex align-items-center rounded group px-3 py-2">
```

Comment:
Please use their `class` to change apply our design rather than adding prefixed classes from bootstrap.